### ISSUE TITLE
Update to latest go-mod-bootstrap with new overrides hyphen to underscore  

### ISSUE THREAD

**Issue by lenny-intel** [edgexfoundry/edgex-go#3393]
Once following PR is merged, the resulting version needs to be used and coordinated merging with merging of App and Device services.

https://github.com/edgexfoundry/go-mod-bootstrap/pull/216

**Comment by lenny-intel** [comment]
still on hold until can coordinate PRs for all services
